Library/Formula/camlp5.rb

```diff
@@ -13,20 +13,41 @@ class Camlp5 < Formula
 
   deprecated_option "strict" => "with-strict"
   option "with-strict", "Compile in strict mode"
+  option "with-doc", "Install the html and info documentation"
+  option "with-tex", "Install the pdf, ps, and tex documentation"
 
   depends_on "ocaml"
+  depends_on "gnu-sed" => :build if build.with?("doc") || build.with?("tex")
+  depends_on :tex => [:build, :optional]
+  depends_on "ghostscript" => :build if build.with?("tex") && !File.exist?("/usr/local/bin/ps2pdf")
 
   def install
-    if build.with? "strict"
-      strictness = "-strict"
-    else
-      strictness = "-transitional"
+    if build.with?("tex")
+      ps2pdf = which("/usr/local/bin/ps2pdf") # known MacTex installation path
```

Comment:
Do you need to call `which` on these? Could you do something like:
```ruby
if build.with?("tex")
  ps2pdf = ("/usr/local/bin/ps2pdf") || (Formula["ghostscript"].opt_bin/"ps2pdf")
  odie "Sorry, can't find your ps2pdf from MacTex's or Homebrew's GhostScript" if ps2pdf.nil?
end
```

I'm still leaning towards thinking this should be a requirement to be honest :confused:. I'm pretty sure in a requirement you'd be able to just call `which("ps2pdf")` as well, which would make it a bit cleaner.